lib/ClangImporter/ClangImporter.cpp

```diff
@@ -563,14 +563,21 @@ getNormalInvocationArguments(std::vector<std::string> &invocationArgStrs,
     invocationArgStrs.push_back("-Xclang");
     invocationArgStrs.push_back("-nostdsysteminc");
   } else {
-    // On Darwin, Clang uses -isysroot to specify the include
-    // system root. On other targets, it seems to use --sysroot.
-    if (triple.isOSDarwin()) {
-      invocationArgStrs.push_back("-isysroot");
+    if (triple.isWindowsMSVCEnvironment()) {
+      llvm::SmallString<261> path;
```

Comment:
Where did this oddly specific number come from?

Reply:
In the Windows API (with some exceptions), [the maximum length for a path is MAX_PATH, which is defined as 260 characters.](https://msdn.microsoft.com/en-us/library/windows/desktop/aa365247(v=vs.85).aspx)

Reply:
261 is `MAX_PATH + 1`, which is pretty much the default size used for path buffers on Windows (the +1 is to ensure null-termination).

Reply:
Can you put that in instead, then?

Reply:
Sure